src/cairo/hello-dojo.md

```diff
@@ -0,0 +1,212 @@
+# Hello Dojo
+
+> This section assumes you have already installed the Dojo toolchain and have a familiarity with Cairo. If you haven't, please refer to the [Getting Started](../getting-started/quick-start.md) section.
+
+## Dojo in 15 minutes
+
+Think of Dojo as an abstraction on top of Cairo (similar to how React is to JS). It allows you to write shorthand commands which expand out at compile time to complex queries. Dojo is built around the well-known architecture called Entity Component System (ECS).
```

Comment:
As the `quick-start` part only handles installing dojoup, maybe we can start from setting up scarb project or using sozo init so that people can follow up full folder structure and then add components and systems?

Reply:
Yeah ok - let me rework and start with `init`